dependencies/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.8"
+services:
+  # A MySQL service as close to production as currently possible
+  # To run it as a standalone container:
+  #   docker run --name mysql-5.7 -e MYSQL_ALLOW_EMPTY_PASSWORD=yes -d -p 27017:3306 mysql:5.7
+  mysql-5.7:
+    image: mysql:5.7
+    ports:
+      - "3306:3306"
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
+    volumes:
+      - mysql-volume:/var/lib/mysql
+
+  # A MongoDB service as close to production as currently possible
+  # To run it as a standalone container:
+  #   docker run --name mongo -d -p 27017:27017 mongo:4.2
+  mongo-4.2:
+    image: mongo:4.2
+    ports:
+      - "27017:27017"
+    volumes:
+      - mongo-volume:/data/db
+
+  # A MS SQL Server service as close to production as currently possible
+  # To run it as a standalone container:
+  #   docker run -e 'ACCEPT_EULA=Y' -e 'SA_PASSWORD=MyS3cr3tPassw0rd' -p 1433:1433 -d --name mssql mcr.microsoft.com/mssql/server:2017-latest
+  mssql-2017:
+    image: mcr.microsoft.com/mssql/server:2017-latest
+    ports:
+      - "1433:1433"
+    environment:
+      ACCEPT_EULA: "Y"
+      SA_PASSWORD: MyS3cr3tPassw0rd
+
+volumes:
```

Comment:
This might need more documentation. If you have a look at the volumes created, when using compose, they are namespaced using the directory they are in. This would lead to confusing results for the inexperienced as data created in the mysql volume for lighthouse would not be available in the mysql volume created by the compose file in crawler.

Reply:
Yes indeed ... it is because of this that I've moved them into this sub folder.  The project name that Docker Compose used was `dependencies` according to when you reviewed this PR, but I have updated this further to use a `.env` file to give the project the name I really wanted, which is now `heron_databases`.  I've updated the README to both explain how to run the docker-compose command for these but also to explain further that they are the same databases in both Crawler and Lighthouse.  I will do the same in Lighthouse so that everything is consistent.

If you're worried about the repos going out of sync (I am a little bit), it is possible to put these three files into a separate repo and then refer to them as sub-modules in the Crawler and Lighthouse repos.  This way, they will both be referring to the same files and when they get updated, you don't have to remember to update in both Crawler and Lighthouse.  When you next work on Crawler or Lighthouse it'll advise you there are new updates to the sub-module.

Reply:
Yes I like the idea of a sub module, we've never used it and it might make a few things easier!